### DISCUSSION TITLE
Getting content to orbit the user

### DISCUSSION THREAD

**Discussion by franbrennan** [google/model-viewer#3412] [Q&A] [answered]
Hi all 👋🙂

I’m maybe just suffering from brain-fog and overlooking something glaringly obvious, but is it possible to set the ‘pivot point’ as being the the user’s own camera/device? So that a model could ‘surround’ the user… as with, say, a gltf of a cloud atlas for example. 

Camera-target/camera-orbit, perhaps?

Any help, thoughts, steers in the right direction, will be much appreciated 🤩

**Comment by elalish** [comment, marked as answer]
Indeed, simply set the radius to zero: `camera-orbit="auto auto 0m"` - *however* that won't work unless you also set the min-radius: `min-camera-orbit="auto auto 0m"`.

**Reply by franbrennan** [reply]
WooHoo! _HUGE_ thanks for that, @elalish … absolutely perfect 🤩😎🚀
